mlops_proj/src/main.py

```
@@ -0,0 +1,135 @@
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+)
+
+import wandb
+import fire
+import numpy as np
+from icecream import ic
+from dotenv import load_dotenv
+
+from src.dataset.watch_log import get_datasets
+from src.dataset.data_loader import data_loader
+from src.model.movie_predictor import movie_predictor, model_save
```

Ruff (0.12.2)

`src.model.movie_predictor.movie_predictor` imported but unused

Remove unused import: `src.model.movie_predictor.movie_predictor`

(F401)